packages/coinstac-ui/app/render/utils/helpers.js

```diff
@@ -1,9 +1,37 @@
 import { indexOf } from 'lodash';
 
-// eslint-disable-next-line import/prefer-default-export
-export const isPipelineOwner = (permissions, owningConsortium) => {
+const COMP_INPUT_NEED_USER_DATA = [
```

Comment:
is this so we know to display a file dialog? Not to harp, but again maybe a HOC/component would be cleaner?

Reply:
Actually looking below I see that may not be the case, there are multiple types that may need mapping, why not just rely on if they are `fulfilled` or not?

Reply:
yeah, you're right. This should rely on the fulfilled.